### ISSUE TITLE
Topic distribution on new document

### ISSUE THREAD

**Issue by Benja1972** [martingerlach/hSBM_Topicmodel#17]
I am wondering, how to get topic distribution of trained model on unseen new document? 
Thank you

**Comment by timhannigan** [comment]
I'm not sure that it's possible with this method, but would defer to the authors on this

**Comment by martingerlach** [comment]
Indeed. currently not possible. we have thought about implementing this feature, but it is not trivial and we havent gotten around to do it yet.
